src/descriptor/mod.rs

```diff
@@ -224,8 +224,28 @@ pub trait PkTranslate<P: MiniscriptKey, Q: MiniscriptKey> {
         Fpk: FnMut(&P) -> Result<Q, E>,
         Fpkh: FnMut(&P::Hash) -> Result<Q::Hash, E>,
         Q: MiniscriptKey;
+
+    /// Calls `translate_pk` with conversion functions that cannot fail
+    fn translate_pk_infallible<Fpk, Fpkh>(
+        &self,
+        mut translatefpk: Fpk,
+        mut translatefpkh: Fpkh,
+    ) -> Self::Output
+    where
+        Fpk: FnMut(&P) -> Q,
+        Fpkh: FnMut(&P::Hash) -> Q::Hash,
+        Q: MiniscriptKey,
+    {
+        self.translate_pk::<_, _, ()>(|pk| Ok(translatefpk(pk)), |pkh| Ok(translatefpkh(pkh)))
+            .expect("infallible translation function")
+    }
 }
 
+// There are some additional convenience functions we could add to  `PkTranslate`
```

Comment:
should remove this comment, it's outdated (I did figure out how to add the convenience functions after all)